distribution/docs/src/main/resources/content/_developing/_devComponents/metacard-type.adoc

```diff
@@ -18,23 +18,26 @@ To define Metacard Types, the definition file must have a `metacardTypes` key in
 }
 ----
 
-The value of `metacardTypes` must be an array of Metacard Type Objects, which are composed of the `type` and `attributes` keys.
+The value of `metacardTypes` must be an array of Metacard Type Objects, which are composed of the `type` (required), `extendsTypes` (optional), and `attributes` (optional) keys.
 
 [source,json]
```

Comment:
Can we add a caption like `.Sample metacardTypes Definition`?

Reply:
Done